pages/install.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/** @var rex_addon $this */
+
+/* setup process */
+
+if (rex_post('install', 'boolean')) {
+
+    $errors = array();
+
+    // step 1/5: select missing packages we need to download
+    $missingPackages = array();
+    $packages = $this->getProperty('setup')['packages'];
+    if (count($packages) > 0) {
+
+        // fetch list of available packages from to redaxo webservice
+        try {
+            $packagesFromInstaller = rex_install_packages::getAddPackages();
+        } catch (rex_functional_exception $e) {
+            $errors[] = $e->getMessage();
+        }
+
+        foreach ($packages as $id => $fileId) {
+
+            $localPackage = rex_package::get($id);
+            if ($localPackage->isSystemPackage()) {
+                continue;
+            }
+
+            $installerPackage = $packagesFromInstaller[$id]['files'][$fileId];
+            if (!$installerPackage) {
+                $errors[] = $this->i18n('package_not_available', $id);
+            }
+
+            if ($localPackage->getVersion() !== $installerPackage['version']) {
```

Comment:
D.h. Sobald es updates von einem der abh. packages im installer gibt funktioniert das hier nicht mehr?

Reply:
D.h. Sollte dieser check auch nach semver gültige kompatible versionen ohne error zulassen?

Reply:
Es wird geprüft, ob die exakt gleiche Version des benötigten AddOns bereits vorhanden ist. Wenn nicht, wird das AddOn im späteren Verlauf runtergeladen und installiert.

Auf Semver wird hier bewusst nicht geprüft aus vielen Gründen, die oben in der Description stehen. Wir arbeiten stattdessen mit explizit definierten AddOn-Versionen, die vorher von uns manuell geprüft werden, dass sie funktionieren.